### PR TITLE
Add "Choose your course" section into In-Company page

### PR DESCRIPTION
**Pull request by thombudel**
screen shot 2017-03-29 at 15 27 11

### CODE REVIEW

File: app/views/shared/_choose_your_course_section.html.erb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      <div class="col-md-4">
+        <div class="card-block">
+          <%= image_tag workshop.image, style: 'height:12em' %>
+          <h3 class="card-title">
+            <%= workshop.title %></h3>
+          <p class="card-text"><%= workshop.description %></p>
+          <a href="#" class="btn btn-primary">Go somewhere</a>
+        </div>
+      </div>
+      <%end%>
```
Comment (miriamtocino) [inline app/views/shared/_choose_your_course_section.html.erb]: `<% end %>`

**Comment by thombudel** [comment]
- Add test
- Render on page

File: app/views/pages/incompany.html.erb [hunk 2]
```diff
@@ -1,2 +1,4 @@
 <h1>Incompany</h1>
-<%= render './shared/get_ready_for_the_day_inCompany' %>
+
+<%= render './shared/choose_your_course' %>
+<%= render './shared/get_ready_for_the_day_(inCompany)' %>
```
Comment (arnoFleming) [inline app/views/pages/incompany.html.erb]: Config your editor so it adds a single newline at the end of the file (see that red thing at the end of this line)

Comment (arnoFleming) [inline app/views/pages/incompany.html.erb]: Do you have a partial with that name? with those `()` signs I mean?

File: app/views/shared/_choose_your_course.html.erb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<section class='jumbotron'>
+  <h3>Choose your course</h3>
+  <p>Get to know our different types of workshops</p>
+
+  <section class="carousel slide" data-ride="carousel" id="postsCarousel">
```
Comment (arnoFleming) [inline app/views/shared/_choose_your_course.html.erb]: why `postsCarousel`?

File: app/views/shared/_choose_your_course.html.erb [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    <div class="row" align="center">
+      <% @workshops.each do |workshop| %>
+      <div class="col-md-4">
+        <div class="card-block">
+          <%= image_tag workshop.image, style: 'height:12em' %>
```
Comment (arnoFleming) [inline app/views/shared/_choose_your_course.html.erb]: don't use inline `style` attributes. We have CSS for that. Give the image_tag a `class: 'workshop-image'` and style that instead

File: app/views/shared/_choose_your_course.html.erb [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+          <h3 class="card-title">
+            <%= workshop.title %></h3>
+          <p class="card-text"><%= workshop.description %></p>
+          <a href="#" class="btn btn-primary">Go somewhere</a>
```
Comment (arnoFleming) [inline app/views/shared/_choose_your_course.html.erb]: don't build links that go nowhere (I don't think we need a link here anyway)

File: app/views/shared/_choose_your_course.html.erb [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+        </div>
+      </div>
+      <% end %>
```
Comment (arnoFleming) [inline app/views/shared/_choose_your_course.html.erb]: nicely done, that iterator!

File: spec/views/pages/incompany.html.erb_spec.rb [hunk 7]
```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe "Render workshops" do
+  let(:workshop1) { create :workshop, title: "Cool Workshop" }
+
+
+  it "shows workshop" do
+
```
Comment (arnoFleming) [inline spec/views/pages/incompany.html.erb_spec.rb]: You're not rendering. You're just testing a model here. So, either move it to `spec/models`  and don't call it 'Render workshops', or create an actual view test (or integration test) and see if you render the right things (that would be a title and a description, from the look of your view)

Comment (arnoFleming) [inline spec/views/pages/incompany.html.erb_spec.rb]: but, possibly you have that model-spec already. Cannot see from this PR

File: app/views/pages/incompany.html.erb [hunk 8]
```diff
@@ -1,2 +1,4 @@
 <h1>Incompany</h1>
-<%= render './shared/get_ready_for_the_day_inCompany' %>
+
+<%= render 'shared/choose_your_course' %>
+<%= render 'shared/get_ready_for_the_day_inCompany' %>
```
Comment (arnoFleming) [inline app/views/pages/incompany.html.erb]: in future: `snake_case_all_filenames`!

File: app/views/shared/_choose_your_course.html.erb [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+    <p>Get to know our different types of workshops</p>
+  </div>
+
+  <section class="carousel slide" data-ride="carousel">
+    <div class="row" align="center">
+      <% @workshops.each do |workshop| %>
+      <div class="col-md-4">
+        <div class="card-block">
+          <%= image_tag workshop.image, class: "workshop-image" %>
+          <h3 class="workshop-title">
```
Comment (arnoFleming) [inline app/views/shared/_choose_your_course.html.erb]: smearing it over 2 lines (instead of 3, or a one-liner) looks icky

**Pull request merged**